managedplugin/docker.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/schollz/progressbar/v3"
 )
 
+var (
+	ErrLoginRequired = fmt.Errorf("login required")
+	ErrTeamRequired  = fmt.Errorf("team required")
```

Comment:
These errors will be enriched by the cli, appending `Hint: do cloudquery <command>`